OpenRA.m

```diff
@@ -121,15 +121,20 @@ - (void)launchGameWithArgs: (NSArray *)gameArgs
 
 	launched = YES;
 
-	// Default values - can be overriden by setting MonoMinVersion and MonoGameExe in Info.plist
+	// Default values - can be overriden by setting certain keys Info.plist
 	NSString *gameName = @"OpenRA.Game.exe";
+	NSString *defaultModId = nil;
 
 	NSDictionary *plist = [[NSBundle mainBundle] infoDictionary];
 	if (plist)
 	{
 		NSString *exeValue = [plist objectForKey:@"MonoGameExe"];
 		if (exeValue && [exeValue length] > 0)
 			gameName = exeValue;
+
+		NSString *defaultModIdValue = [plist objectForKey:@"DefaultModId"];
```

Comment:
I've come to the conclusion that our only feasible way forwards on the `OpenRA.Game.exe` side is to make `Game.Mod` a required commandline parameter and remove the problematic `Mod` settings.yaml key that is causing all our problems.  Once that is done `Engine.DefaultMod` will be redundant and can be removed.

Looking ahead to this change, can you please drop the "default" from the plist key and variable names?